docs/source/db_facts.rst

```diff
@@ -0,0 +1,29 @@
+db\_facts package
+=================
+
+Submodules
+----------
+
+db\_facts.base64\_jinja\_context module
+---------------------------------------
+
+.. automodule:: db_facts.base64_jinja_context
+   :members:
+   :undoc-members:
+   :show-inheritance:
+
+db\_facts.env\_jinja\_context module
+------------------------------------
+
+.. automodule:: db_facts.env_jinja_context
+   :members:
+   :undoc-members:
+   :show-inheritance:
+
+Module contents
+---------------
+
+.. automodule:: db_facts
+   :members:
+   :undoc-members:
+   :show-inheritance:
```

Comment:
This is hand-tuned to show the bits we want to document - the public API as well as the details about the provided jinja contexts.  Having the rest marked private with a leading underscore may allow us to have this file be auto-generated so we won't have to tweak it if more public API is added in the future.